src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -41,22 +40,25 @@
     private final CountryRepository countryRepository;
     private final CityRepository cityRepository;
 
-    @Value("${file.path}")
-    private String filePath;
+    private final FileService fileService;
 
     public FormFullDTO process(FormRegisterDTO form, MultipartFile file) {
 
         MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
 
         if (file != null) {
-
-            form.setFilePath(file.getOriginalFilename());
+            String fileURL = null;
+            try {
+                fileURL = fileService.upload(file.getBytes(),file.getOriginalFilename());
+            } catch (IOException e) {
+                throw new FileNotUploadException(e.getMessage());
+            }
+
+            form.setFilePath(fileURL);
```

Comment:
Move this line inside try block and remove line 50

Reply:
Done